pkg/status/status_server.go

```diff
@@ -50,11 +50,11 @@ const (
 	patternHelp               = "/help"
 	patternOptions            = "/options"
 	patternVersion            = "/version"
-	patternBpfAdsMaps         = "/debug/config_dump/bpf/ads"
-	patternBpfWorkloadMaps    = "/debug/config_dump/bpf/workload"
+	patternBpfAdsMaps         = "/debug/config_dump/bpf/kernel-native"
+	patternBpfWorkloadMaps    = "/debug/config_dump/bpf/dual-engine"
 	configDumpPrefix          = "/debug/config_dump"
-	patternConfigDumpAds      = configDumpPrefix + "/ads"
-	patternConfigDumpWorkload = configDumpPrefix + "/workload"
+	patternConfigDumpAds      = configDumpPrefix + "/kernel-native"
+	patternConfigDumpWorkload = configDumpPrefix + "/dual-engine"
```

Comment:
The error should not be exposed directly if kernel-native mode is used in duel-engine mode. 
The user should be prompted that cannot use the kernel-native command dump map in `duel-engine` mode.
So I recommend optimising the error message.

Reply:
Ok, I will optimise the error message